checklists/aks_checklist.en.json

```diff
@@ -467,11 +467,19 @@
     {
       "category": "Operations",
       "subcategory": "Compliance",
-      "text": "Use kured for Linux node upgrades",
+      "text": "Use kured for Linux node upgrades in case you are not using node-image upgrade",
       "guid": "6f7c4c0d-4e51-4464-ad24-57ed67138b82",
       "severity": "High",
       "link": "https://docs.microsoft.com/azure/aks/node-updates-kured"
     },
+    {

```

Comment:
This check is very similar to e189c599-df0d-45a7-9dd4-ce32c1881370 ("Have a regular process to upgrade the cluster periodically (quarterly, for example)"). How are they different?